packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -158,39 +161,36 @@ export const ChatListItem = ({ chat, myId, isSelected, isRead }: IMgtChatListIte
   // Copied and modified from the sample ChatItem.tsx
   // Determines the title in the case of 1:1 and self chats
   // Self Chats are not possible, however, 1:1 chats with a bot will show no other members other than self.
-  const inferTitle = (chatObj: Chat) => {
-    if (myId && chatObj.chatType === 'oneOnOne' && chatObj.members) {
-      const other = chatObj.members.find(m => (m as AadUserConversationMember).userId !== myId);
-      const me = chatObj.members.find(m => (m as AadUserConversationMember).userId === myId);
-      const application = chatObj.lastMessagePreview?.from?.application as TeamworkApplicationIdentity;
-      // if there is no other member, return the application display name
-      if (other) {
-        return `${other?.displayName || (other as AadUserConversationMember)?.email || other?.id}`;
-      } else if (application && me) {
-        return `${application?.displayName}` || `${application?.id}`;
-      }
-    }
-    if (chatObj.chatType === 'group' && chatObj.members) {
-      const others = chatObj.members.filter(m => (m as AadUserConversationMember).userId !== myId);
-      // if there are 3 or less members, display all members' first names
-      if (chatObj.members.length <= 3) {
-        return (
-          chatObj.topic ||
-          others.map(m => (m as AadUserConversationMember).displayName?.split(' ')[0]).join(', ') ||
-          chatObj.chatType
-        );
-        // if there are more than 3 members, display the first 3 members' first names and a count of the remaining members
-      } else if (chatObj.members.length > 3) {
-        let firstThreeMembersSlice = others.slice(0, 3);
-        let remainingMembersCount = chatObj.members.length - 3;
-        let groupMembersString =
-          firstThreeMembersSlice.map(m => (m as AadUserConversationMember).displayName?.split(' ')[0]).join(', ') +
-          ' +' +
-          remainingMembersCount;
-        return chatObj.topic || groupMembersString || chatObj.chatType;
-      }
+  const inferTitle = (chat: Chat) => {
```

Comment:
just a suggestion but it seemed a good piece of code to unit test since we have a lot of conditions - maybe to add to their existing unit test list?

Reply:
As far as I know all those unit tests are dead until we get the go-ahead from Manish.

Reply:
what I mean is to maybe abstract the code as a service for unit testing purposes as we are able to test ts file today, just not component. I am also fine if we want to not have that added complexity of abstracting this out. Just suggestion since this seemed to have a lot of conditions to test for.

Reply:
I agree with the assertion that it should be tested. But rather than figure out how we would test that at this stage, I want to delay until after Manish finishes his assessment and then we can do that.